src/plugins/auto/src/auto_schedule.cpp

```diff
@@ -174,6 +175,11 @@ void AutoSchedule::init() {
                     m_compile_context[CPU].m_device_info = *cpu_iter;
                     m_compile_context[CPU].m_device_info.config[ov::hint::performance_mode.name()] =
                         ov::hint::PerformanceMode::LATENCY;
+                    if (m_compile_context[ACTUALDEVICE].m_device_info.config.count(ov::cache_dir.name()) &&
+                        (m_context->m_startup_fallback || m_context->m_runtime_fallback)) {
+                        m_compile_context[CPU].m_device_info.config[ov::cache_dir.name()] = "";
+                        LOG_INFO_TAG("Clear cache dir setting for CPU accelerator");
+                    }
```

Comment:
is it possible to move all cache related customization/operations to one function/one single entry? 
we have cache operations a few lines above this change, and then some other cache operation here....

Reply:
will implement.

Reply:
Updated.